schemas/definitions.json

```diff
@@ -269,43 +272,52 @@
         "guidelines": [
           ["Must start with a capital letter."],
           ["Must end with a period."],
-          ["Should describe the data product, not the snapshot (i.e. the subset of data we extract from the data product)."]
+          ["Must not mention other metadata fields like `producer` or `version_producer`.", {"type": "exceptions", "value": ["Other metadata fields are crucial in the description of the data product."]}],
```

Comment:
would modify

"Other metadata fields are crucial in the description of the data product."

to

"These other metadata fields are crucial in the description of the data product."